packages/client/lib/OpenID4VCIClient.ts

```diff
@@ -183,21 +161,33 @@ export class OpenID4VCIClient {
     const parEndpoint: string = this._endpointMetadata.credentialIssuerMetadata.pushed_authorization_request_endpoint;
 
     // add 'openid' scope if not present
-    if (scope && !scope.includes('openid')) {
-      scope = `openid ${scope}`;
+    if (!scope?.includes('openid')) {
+      scope = ['openid', scope].filter((s) => !!s).join(' ');
     }
 
-    //fixme: handle this for v11
-    const queryObj: AuthorizationRequestV1_0_09 = {
+    const queryObj: { [key: string]: string } = {
       response_type: ResponseType.AUTH_CODE,
-      client_id: clientId,
+      client_id: this.clientId || '',
```

Comment:
Not entirely sure about using empty strings here

Reply:
Yes, I agree. I have updated the logic to add `client_id` parameter only if it was provided during initialization.